src/test/java/org/apache/commons/io/output/DeferredFileOutputStreamTest.java

```diff
@@ -365,4 +370,55 @@ private void verifyResultFile(final File testFile) {
             fail("Unexpected IOException");
         }
     }
+
+    /**
+     * Tests the case where the amount of data falls below the threshold, and is therefore confined to memory.
+     * Testing the getInputStream() method.
+     */
+    @ParameterizedTest(name = "initialBufferSize = {0}")
+    @MethodSource("data")
+    public void testBelowThresholdGetInputStream(final int initialBufferSize) throws IOException {
+        final DeferredFileOutputStream dfos = new DeferredFileOutputStream(testBytes.length + 42, initialBufferSize,
+            null);
+        try {
+            dfos.write(testBytes, 0, testBytes.length);
+            dfos.close();
+        } catch (final IOException e) {
+            fail("Unexpected IOException");
+        }
+        assertTrue(dfos.isInMemory());
+
+        try(InputStream is = dfos.toInputStream()) {
+            final byte[] resultBytes = IOUtils.toByteArray(is);
+            assertArrayEquals(testBytes, resultBytes);
+        }
+    }
+
+    /**
+     * Tests the case where the amount of data exceeds the threshold, and is therefore written to disk. The actual data
+     * written to disk is verified, as is the file itself.
+     * Testing the getInputStream() method.
+     */
+    @ParameterizedTest(name = "initialBufferSize = {0}")
+    @MethodSource("data")
```

Comment:
See other comments, which apply here as well. Use final in method parmeter.

Reply:
Done.